src/common/databaseFilters.ts

```diff
@@ -4,46 +4,55 @@ import { DbClientRow, DbParcelRow } from "@/databaseUtils";
 import { parcelsPageDeletedClientDisplayName } from "@/app/parcels/parcelsTable/format";
 
 export const fullNameSearch = <DbData extends DbClientRow | DbParcelRow>(
-    columnLabel: Extract<keyof DbData, "full_name" | "client_full_name">
+    fullNameColumnLabel: Extract<keyof DbData, "full_name" | "client_full_name">,
+    clientIsActiveColumnLabel: Extract<keyof DbData, "is_active" | "client_is_active">
 ): ServerSideFilterMethod<DbData, string> => {
     return (query, state) => {
         if (state === "") {
             return query;
         }
         if (parcelsPageDeletedClientDisplayName.toLowerCase().includes(state.toLowerCase())) {
-            return query.or(`${columnLabel}.ilike.%${state}%, ${columnLabel}.is.null`);
+            return query.or(
+                `${fullNameColumnLabel}.ilike.%${state}%, ${clientIsActiveColumnLabel}.eq.false`
+            );
         }
-        return query.ilike(`${columnLabel}`, `%${state}%`);
+        return query.ilike(`${fullNameColumnLabel}`, `%${state}%`);
```

Comment:
I'm not sure which line you're referring to but line 19 is just a variable being renamed. lines 15-17 means when we search for a parcel with name ("Deleted Client") we look for parcels in the DB with clients with `is_active = false` rather than parcels with a null client name